projects/core/src/user/store/reducers/index.ts

```diff
@@ -1,35 +1,39 @@
 import { InjectionToken, Provider } from '@angular/core';
-
-import { ActionReducerMap, MetaReducer, ActionReducer } from '@ngrx/store';
-
 import {
-  UserState,
-  USER_ORDERS,
-  USER_PAYMENT_METHODS,
-  USER_ADDRESSES,
-} from '../user-state';
+  ActionReducer,
+  ActionReducerMap,
+  combineReducers,
+  MetaReducer,
+} from '@ngrx/store';
 import { LOGOUT } from '../../../auth/index';
 import {
-  PaymentDetails,
-  OrderHistoryList,
   Address,
+  OrderHistoryList,
+  PaymentDetails,
 } from '../../../occ/occ-models/occ.models';
 import { loaderReducer } from '../../../state/utils/loader/loader.reducer';
-
+import {
+  UserState,
+  USER_ADDRESSES,
+  USER_ORDERS,
+  USER_PAYMENT_METHODS,
+} from '../user-state';
 import * as fromBillingCountriesReducer from './billing-countries.reducer';
 import * as fromDeliveryCountries from './delivery-countries.reducer';
 import * as fromOrderDetailsReducer from './order-details.reducer';
 import * as fromPaymentReducer from './payment-methods.reducer';
 import * as fromRegionsReducer from './regions.reducer';
+import * as fromResetPasswordReducer from './reset-password.reducer';
 import * as fromTitlesReducer from './titles.reducer';
 import * as fromAddressesReducer from './user-addresses.reducer';
 import * as fromUserDetailsReducer from './user-details.reducer';
 import * as fromUserOrdersReducer from './user-orders.reducer';
-import * as fromResetPasswordReducer from './reset-password.reducer';
 
 export function getReducers(): ActionReducerMap<UserState> {
   return {
-    account: fromUserDetailsReducer.reducer,
+    account: combineReducers({
+      details: fromUserDetailsReducer.reducer,
+    }),
```

Comment:
Do we still need to update the user state considering we store the process state elsewhere?

Reply:
Yes, we're merging the state locally.